package.json

```diff
@@ -228,6 +228,7 @@
     "screenfull": "4.0.0",
     "shpjs": "3.4.2",
     "stickybits": "3.6.6",
+    "style-it": "^2.1.4",
```

Comment:
We could remove this library if we use the style and dangerouslySetInnerHTML approach